sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -128,6 +128,14 @@ def set_power_override(self, power_override, power_set):
     def get_eeprom_path(self):
         raise NotImplementedError
 
+    def get_lpmode(self):
+        api = self.get_xcvr_api()
+        return api.get_lpmode() if api is not None else None
+
+    def set_lpmode(self, lpmode):
+        api = self.get_xcvr_api()
+        return api.set_lp_mode(lpmode) if api is not None else None
```

Comment:
the return type should match with the return type mentioned in sfp_base.py for this api